domains/prtksunal.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "prateekmedia",
+        "email": "prtksunal@gmail.com"
+    },
+    "record": {
+        "CNAME": "prateekmedia.github.io"
```

Comment:
CNAME just redirects back to our domain. Please display content.

Reply:
Done!